sync/wikiman.py

```diff
@@ -47,19 +47,23 @@ def make_artifact_entry(details: Dict[str, str]) -> Dict[str, Any]:
     else:
         raise ValueError(f"'repoName' or 'url' key not specified for '{name}' in '{SOURCE_YAMLFILE}'")
 
-    return {
+    entry = {
         'name': name,
         'artifactUrl': artifact_url,
         'artifactLevel': 1,
         'destination': details['destination'],
+        'remove' : remove_from_all + details.get('remove', []),
```

Comment:
I don't like that `remove_from_all`, a variable defined outside of the method, is used inside the method. It's also not needed.

I suggest that this becomes:
```suggestion
        'remove' : details['remove'],
```
and see below for the other suggestion.

Reply:
Done differntly in https://github.com/wbstack/mediawiki/pull/185/commits/2e886a9deb0af2830e28c3a84a6cd163937b21e3